interactors/nonceHandlerV2/nonceTransactionsHandler.go

```diff
@@ -17,6 +17,13 @@ const minimumIntervalToResend = time.Second
 
 var log = logger.GetOrCreate("elrond-sdk-erdgo/interactors/nonceHandlerV2")
 
+// ArgsNonceTransactionsHandlerV2 is the argument DTO for a pair
+type ArgsNonceTransactionsHandlerV2 struct {
+	Proxy                         interactors.Proxy
+	IntervalToResend              time.Duration
+	CreateAddressNonceHandlerFunc func(proxy interactors.Proxy, address core.AddressHandler) (interactors.AddressNonceHandler, error)
```

Comment:
Personally, I do not like this approach of providing this handler function
The better way to do this is to apply the abstract factory pattern:
1. create 2 small structures that implement the same interface "AddressNonceHandlerCreator", each defining the following method:
```GO
Create(proxy interactors.Proxy, address core.AddressHandler) (interactors.AddressNonceHandler, error)
```
2. use the interface in this Args struct, calling the Create method where needed
3. create unit tests for all implementations. Surprisingly, those structs can have 100% code coverage